sunscreen_zkp_backend/src/jit.rs

```diff
@@ -482,6 +494,11 @@ where
         v.add_session(session.unwrap());
     }
 
+    #[cfg(feature = "debugger")]
+    if let Some(id) = unsatisfied_constraint {
+        return Err(Error::UnsatisfiableConstraint(id));
+    }
+
```

Comment:
How do the users get the node ouputs where we stuck the failed constraints?

Reply:
If you mean the outputs vs the constraint, they should just click on the nodes leading into the constraint node.

Reply:
I mean, this function is adding problematic nodes to the `node_outputs` variable, but then at the end if there were any problematic nodes, it just returns an `Err`. So how are you getting data out of this function?

Reply:
On like 483 the `node_outputs` variable gets put into `session.run_data`, and we add the session before we return the `Err`.

Reply:
Ah I see, thanks for clarifying!